study/src/test/java/di/stage3/context/DIContainer.java

```diff
@@ -9,12 +15,36 @@ class DIContainer {
 
     private final Set<Object> beans;
 
-    public DIContainer(final Set<Class<?>> classes) {
-        this.beans = Set.of();
+    public DIContainer(Set<Class<?>> classes) {
+        this.beans = createBeans(classes);
+        this.beans.forEach(this::assignFields);
+    }
+
+    private Set<Object> createBeans(Set<Class<?>> classes) {
+        return classes.stream()
+                .map(FunctionWrapper.apply(Class::getDeclaredConstructor))
+                .peek(constructor -> constructor.setAccessible(true))
+                .map(FunctionWrapper.apply(Constructor::newInstance))
+                .collect(Collectors.toSet());
+    }
+
+    private void assignFields(Object bean) {
+        Arrays.stream(bean.getClass().getDeclaredFields())
+                .peek(field -> field.setAccessible(true))
```

Comment:
[제안]

가능하면 setAccessible에 대한 설정도 다시 닫아주는 것이 좋습니다!

```suggestion
                .peek(field -> field.setAccessible(true))
                .peek(ConsumerWrapper.accept(field-> assingField(field, bean))
                .peek(field -> field.setAccessible(false))
                
```

Reply:
[반영] [8ca4038](https://github.com/woowacourse/java-mvc/pull/835/commits/8ca40383887607ac5c4b3c3467ee360195c3b551)